lnwallet/channel.go

```diff
@@ -1095,8 +1096,13 @@ func (u *updateLog) appendUpdate(pd *PaymentDescriptor) {
 // restoreUpdate appends a new update to the tip of the updateLog. The entry is
 // also added to index accordingly. This function differs from appendUpdate in
 // that it won't increment the log index counter.
-func (u *updateLog) restoreUpdate(pd *PaymentDescriptor) {
+func (u *updateLog) restoreUpdate(pd *PaymentDescriptor) bool {
```

Comment:
Should modify `godoc` comment to explain the new return value both here and for the other method.